util/localconfig/localconfig.go

```diff
@@ -262,3 +263,11 @@ func DefaultLocalConfigPath() (string, error) {
 	}
 	return path.Join(dir, "config"), nil
 }
+
+func GetUsername(subject string) string {
+	parts := strings.Split(subject, ":")
+	if len(parts) > 0 {
+		return parts[0]
+	}
+	return subject
```

Comment:
Since `strings.Split()` always returns an array with at least one element, this can be simplified to:

```go
return strings.Split(subject, ":")[0]
```

From https://golang.org/pkg/strings/#Split
>  If s does not contain sep and sep is not empty, Split returns a slice of length 1 whose only element is s.